### PR TITLE
Checkpoint2

### PR DESCRIPTION
**Pull request by jophinjohn**
2 Rails Serializers

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -12,7 +12,8 @@ gem 'devise'
 gem 'pundit'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 
-gem 'rails', '4.2.4'
+
+gem 'rails', '5.0.0.beta1.1'
```
Comment (drobtravels) [inline Gemfile]: I would recommend downgrading to 4.2.5  

You should probably be ok to just change it and bundle install.  If it causes problems though you can go back to 5.

File: db/migrate/20160129090541_remove_type_from_lists.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class RemoveTypeFromLists < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :lists, :type, :string
```
Comment (drobtravels) [inline db/migrate/20160129090541_remove_type_from_lists.rb:3]: Ah good call.  You may have discovered.. but the column named `type` has a special meaning in ActiveRecord and would have caused some odd errors.  There is a warning about this in the [Active Record Basics Guide](http://guides.rubyonrails.org/active_record_basics.html#schema-conventions)

File: test/models/item_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ItemTest < ActiveSupport::TestCase
```
Comment (drobtravels) [inline test/models/item_test.rb:3]: Please delete these `_test` files if you aren't going to use them.

Also, if you do want tests,  you should use Rspec like we did for other projects.  Everything under the folder `test` is for a different testing framework `ActiveSupport::Test` which is included in Rails by default but not as commonly used

**Comment by jophinjohn** [comment]
Closed

**Pull request closed without merging**